CHANGELOG.md

```diff
@@ -171,31 +171,31 @@ Color=#ccffcc
 - improved applog
 - improved agent events
 - fixed category and state display for history
-- unifyed catalog cab and update download
+- unified catalog cab and update download
 - improved custom update downloader
 
 ## [0.3] - 2018-09-02
 ### Added
 - System tray Icon
 - Auto Start
 - UAC bypass for administrator users
-- added warning if running without window supdate service enabled
+- added warning if running without windows update service enabled

```

Comment:
`windows update` should be `Windows Update`, since it's referring to the OS/service, not words of generic purpose